src/Access/AccessControlManager.cpp

```diff
@@ -295,6 +308,12 @@ void AccessControlManager::addStoragesFromUserDirectoriesConfig(
             bool readonly = config.getBool(prefix + ".readonly", false);
             addDiskStorage(name, path, readonly);
         }
+        else if (type == LDAPAccessStorage::STORAGE_TYPE)
+        {
+            if (bracket_pos != String::npos)
+                throw Exception("Duplicate storage type '" + type + "' at " + prefix + " in config", ErrorCodes::EXCESSIVE_ELEMENT_IN_CONFIG);
```

Comment:
The checks for duplications are normally performed inside corresponding `add*Storage()` functions.
So just for consistence please move this code.

Reply:
Removed the checks, not relevant anymore.